libc/include/wchar.yaml

```diff
@@ -6,6 +6,7 @@ macros:
 types:
   - type_name: FILE
   - type_name: size_t
+  - type_name: struct_tm
```

Comment:
This list uses the actual name, i.e. `struct tm`, not the derived file name form.

This could use a TODO comment that the ones only here because of missing function signatures that would imply them should be removed when the full suite of signatures is added to the YAML file.  Unless a standard specifically says that the header exports the type, then we should not list it here as a first-class export but only let it be implicit from references in function signatures.

Reply:
Ah, I misremembered the auto generation from signatures vs this field.
I think we can leave this as it is for now.
I would like to change the hdrgen semantics a bit here, but that will part of larger cleanups that can wait.